CHANGELOG.md

```diff
@@ -1,5 +1,40 @@
 # Changelog
 
+## v1.12.0 (03.03.2021)
+
+### Enhancements
+
+#### :gear: Server-Side Web Assets Caching ([testcafe-hammerhead/#863](https://github.com/DevExpress/testcafe-hammerhead/issues/863))
+
+TestCafe's proxy server can now cache web assets (like images, scripts and videos). When TestCafe revisits a website, it loads assets from this cache, saving time on repetetive network requests.
+
+To enable server-side caching, use any of the following:
+
+* [the `--cache` CLI flag](./docs/articles/documentation/reference/command-line-interface.md#--cache)
+* [the `cache` configuration file property](./docs/articles/documentation/reference/configuration-file.md#cache)
+* [the `cache` runner option](./docs/articles/documentation/reference/testcafe-api/runner/run.md)
+
+#### :gear: Asynchronous Predicate Function
+
+The following TestCafe entities now support **asynchronous** predicate functions:
```

Comment:
You usually don't have to use the vague term 'entity' when speaking about code entities. Here, you can call them 'request hooks', can't you?

*The following request hooks now support...*